plumbing/transport/internal/common/common.go

```diff
@@ -146,13 +147,21 @@ func (c *client) listenFirstError(r io.Reader) chan string {
 		return nil
 	}
 
+	skipPattern := regexp.MustCompile("^remote: (=)*(\r\n|\n|\r){0,1}$")
```

Comment:
Please move this outside the func, so it is compiled only once instead of at every call.